### PR TITLE
Pass API Key through headers, not param

### PR DESCRIPTION
**Pull request by BryceStevenWilley**
Params are printed in the uWSGI logs on docassemble, potentially leaking the API key to other developers.

### CODE REVIEW

File: lib/docassemble/docassemble_api_interface.js [hunk 1]
```diff
@@ -222,7 +222,7 @@ da_i.loop_to_create_project = async function( creation_options ) {
 
     } catch ( error ) {
       // TODO: Make a `normalize_ajax_responses_and_errors()`?
-      let response = error.response;
+      let response = error;
```
Comment (plocket) [inline lib/docassemble/docassemble_api_interface.js:225]: This is unrelated, right? A different bug?

**Review by plocket — APPROVED** [review]
Great catch and looks good, thank you!

File: lib/docassemble/docassemble_api_interface.js (hunk 1 above)
Comment (BryceStevenWilley) [reply lib/docassemble/docassemble_api_interface.js:225]: Yes, different bug but I did need to fix it to test. I think it's related to those 400 errors we've been seeing with Caroline and elsewhere recently.
